modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/portlet/action/UpdateTranslationMVCActionCommand.java

```diff
@@ -87,15 +95,46 @@ protected void doProcessAction(
 					infoItemReference.getClassName(),
 					ClassPKInfoItemIdentifier.INFO_ITEM_SERVICE_FILTER);
 
+			double version = ParamUtil.getDouble(actionRequest, "version");
+
+			Object infoItem = infoItemObjectProvider.getInfoItem(
+				new ClassPKInfoItemIdentifier(classPK));
```

Comment:
@Mikellorza, I know that this was so before your changes, but we could change to use infoItemReference.getInfoItemIdentifier() instead of  new ClassPKInfoItemIdentifier(classPK) here since we already have the object

Reply:
yes, you are right, I changed it, thanks @jkappler